gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js

```diff
@@ -285,6 +285,34 @@ gpii.test.cloudBased.oauth2.verifySecuredSettingsResponse = function (body, secu
     jqUnit.assertDeepEq("Expected settings returned from flowManager", expected, response);
 };
 
+gpii.test.cloudBased.oauth2.sendResourceOwnerGpiiTokenAccessTokenRequest = function (accessTokenRequest, options) {
+    var formBody = {
+        grant_type: "password",
+        client_id: options.client_id,
+        client_secret: options.client_secret,
+        username: options.username,
+        password: options.password
+    };
+
+    gpii.test.cloudBased.oauth2.sendRequest(accessTokenRequest, options, formBody, "accessTokenForm");
+};
+
+gpii.test.cloudBased.oauth2.verifyResourceOwnerGpiiTokenAccessTokenInResponse = function (body, accessTokenRequest) {
+    var response = gpii.test.verifyJSONResponse(body, accessTokenRequest);
+    gpii.test.cloudBased.oauth2.verifyFieldInResponse(response, accessTokenRequest, "oauth2.verifyUntrustedSettingsAccessTokenInResponse", "access_token");
+    gpii.test.cloudBased.oauth2.verifyFieldInResponse(response, accessTokenRequest, "oauth2.verifyUntrustedSettingsAccessTokenInResponse", "expiresIn");
```

Comment:
I think that the 3rd argument in these 2 calls to `gpii.test.cloudBased.oauth2.verifyFieldInResponse` needs to be updated to `"oauth2.verifyResourceOwnerGpiiTokenAccessTokenInResponse"`.